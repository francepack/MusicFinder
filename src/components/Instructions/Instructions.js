import React from 'react'

export const Instructions = () => {
  return(
    <div className='instructions'>
      <p>Type in the name of your favorite band above.</p>
      <p>Submit to find similar bands and descriptions of that band.</p>
      <p>Click on any result to search for events that may be of interest.</p>
    </div>
  )
}