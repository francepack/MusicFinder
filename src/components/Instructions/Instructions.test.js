import React from 'react'
import { Instructions } from './Instructions'
import { shallow } from 'enzyme'

describe('Instructions', () => {
  describe('snapshot', () => {
    it('should match snapshot', () => {
      let wrapper = shallow(
        <Instructions />
      )
      expect(wrapper).toMatchSnapshot()
    })
  })
})