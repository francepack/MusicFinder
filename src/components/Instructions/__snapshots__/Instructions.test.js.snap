// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Instructions snapshot should match snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Instructions />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <p>
          Type in the name of your favorite band above.
        </p>,
        <p>
          Submit to find similar bands and descriptions of that band.
        </p>,
        <p>
          Click on any result to search for events that may be of interest.
        </p>,
      ],
      "className": "instructions",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Type in the name of your favorite band above.",
        },
        "ref": null,
        "rendered": "Type in the name of your favorite band above.",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Submit to find similar bands and descriptions of that band.",
        },
        "ref": null,
        "rendered": "Submit to find similar bands and descriptions of that band.",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Click on any result to search for events that may be of interest.",
        },
        "ref": null,
        "rendered": "Click on any result to search for events that may be of interest.",
        "type": "p",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <p>
            Type in the name of your favorite band above.
          </p>,
          <p>
            Submit to find similar bands and descriptions of that band.
          </p>,
          <p>
            Click on any result to search for events that may be of interest.
          </p>,
        ],
        "className": "instructions",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Type in the name of your favorite band above.",
          },
          "ref": null,
          "rendered": "Type in the name of your favorite band above.",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Submit to find similar bands and descriptions of that band.",
          },
          "ref": null,
          "rendered": "Submit to find similar bands and descriptions of that band.",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Click on any result to search for events that may be of interest.",
          },
          "ref": null,
          "rendered": "Click on any result to search for events that may be of interest.",
          "type": "p",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
